src/types/importCandidate.ts

```diff
@@ -0,0 +1,56 @@
+import { Organization } from './organization.types';
+import { JournalPublicationInstance } from './publication_types/journalRegistration.types';
+import { DegreePublicationInstance } from './publication_types/degreeRegistration.types';
+import { BookPublicationInstance } from './publication_types/bookRegistration.types';
+import { ReportPublicationInstance } from './publication_types/reportRegistration.types';
+import { ChapterPublicationInstance } from './publication_types/chapterRegistration.types';
+import { PresentationPublicationInstance } from './publication_types/presentationRegistration.types';
+import { ArtisticPublicationInstance } from './publication_types/artisticRegistration.types';
+import { MediaContributionPeriodicalPublicationInstance } from './publication_types/mediaContributionRegistration.types';
+import { ResearchDataPublicationInstance } from './publication_types/researchDataRegistration.types';
+import { MapPublicationInstance } from './publication_types/otherRegistration.types';
+import { ExhibitionPublicationInstance } from './publication_types/exhibitionContent.types';
+
+export enum ImportStatus {
+  Imported = 'IMPORTED',
+  NotImported = 'NOT_IMPORTED',
+  NotApplicable = 'NOT_APPLICABLE',
+}
+
+export interface Publisher {
+  id: string;
+  name: string;
+}
+
+export interface Journal {
+  id: string;
+}
```

Comment:
Antar disse er basert på typene vi har fra før på `Journal` og `Publisher` (`registration.types.ts`)? 

I så fall kan disse fjernes, så kan vi gjenbruke de i stedet, hvor dette vil være det samme og kan brukes i stedet for `ImportCandidate`:
```suggestion
```
`Pick<Publisher, 'id' | 'name'>`
`Pick<Journal, 'id'>`